helix-term/Cargo.toml

```diff
@@ -55,5 +53,16 @@ toml = "0.5"
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 
+tracing = "0.1.26"
+tracing-appender = "0.1.2"
+tracing-tracy = { version = "0.7.0", optional = true }
```

Comment:
We don't need `tracing-tracy` or `tracing-flame`, imo.

Reply:
wasn't it in your pr?

Reply:
Yeah, I took it out but I never pushed the change.

Reply:
what was your reason for taking it out? Here it is feature gated so it should be okay to add it

Reply:
I don't think it's not something that'll be used very often (it's troublesome to compile the client on platforms other than Windows). Even if it's optional, I think it's just sanitary to prune what we don't need.

Reply:
I had no problem compiling tracy on linux even though I had to compile the source. I think it is pretty useful because it supports real time profiling while flamegraphs do not. It also supports some form of observability and it seems to have more features than flamegraphs.

Reply:
@archseer had problems, afaik.

Reply:
It's fine to keep given that it's feature gated.